javascript/chronometer.js

```diff
@@ -1,36 +1,54 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0;
+    this.intervalId = null;
     // ... your code goes here
   }
 
   start(callback) {
+    this.intervalId = setInterval(()=>{this.currentTime ++},1000)
     // ... your code goes here
   }
 
   getMinutes() {
+    return Math.floor(this.currentTime / 60);
     // ... your code goes here
   }
 
   getSeconds() {
+    return Math.floor(this.currentTime % 60)
     // ... your code goes here
   }
 
   computeTwoDigitNumber(value) {
+    if(value < 10){
+      return `0${value}`
+    }
+    else{
+      return `${value}`
+    }
     // ... your code goes here
   }
 
   stop() {
+    clearInterval(this.intervalId)
     // ... your code goes here
   }
 
   reset() {
+    this.currentTime = 0
     // ... your code goes here
   }
 
-  split() {
-    // ... your code goes here
+  split() {if (this.getMinutes() < 10 || this.getSeconds < 10) {
+    return (
+      this.computeTwoDigitNumber(this.getMinutes()) + ':' + this.computeTwoDigitNumber(this.getSeconds())
+    );
+  } else {
+    return this.getMinutes() + ':' + this.getSeconds();
   }
```

Comment:
weil du alles wichtige schon in computeTwoDigitNumber definiert hast, hättest du es so machen können:
```

  split() {
    let minutes = this.computeTwoDigitNumber(this.getMinutes());
    let seconds = this.computeTwoDigitNumber(this.getSeconds());

    return `${minutes}:${seconds}`;
  }
```